src/lib.rs

```diff
@@ -87,6 +124,39 @@ pub enum TearingEffect {
     HorizontalAndVertical,
 }
 
+///
+/// Options for displays used on initialization
+///
+#[derive(Copy, Clone, Default)]
+pub struct DisplayOptions {
+    /// Initial display orientation (without inverts)
+    pub orientation: Orientation,
+    /// Set to make display vertical refresh bottom to top
+    pub invert_vertical_refresh: bool,
+    /// Set to make display use BGR instead of RGB pixel format
+    pub rgb_to_bgr: bool,
```

Comment:
This doesn't fit the st7735s as it needs to be true but then it sounds like we're using BGR when it is supposed to be RGB. I am assuming that the st7789 is the same.

Reply:
I suggest instead of a bool we use an `Enum { BGB, RGB}` here.

Reply:
`Model::init` should take care of the inverting the bit logic here. The display options can stay out of it, but I agree about the enum.